src/main/java/org/micks/DiscGolfApplication/DiscGolfDbConnection.java

```diff
@@ -0,0 +1,19 @@
+package org.micks.DiscGolfApplication;
```

Comment:
Maybe not related to this PR, but do we want org.micks.* package?

Reply:
We can think of it 🙂